src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/Commands.xml

```diff
@@ -3,49 +3,82 @@
      Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information. -->
 <Commands>
   <Verb Name="add">
-    <Noun Name="source">
+    <Noun Name="source" Version="3.1.200"
+       Description="adds a new package source to your NuGet configuration files."
+       >
       <Argument Name="Source" LongName="PackageSourcePath" Help="SourcesCommandSourceDescription" />
       <SingleValueOption Name="name" Shortcut="n" Help="SourcesCommandNameDescription"/>
-      <SingleValueOption Name="username" Shortcut="u" Help="SourcesCommandUserNameDescription"/>
+      <SingleValueOption Name="username" Shortcut="u" Help="SourcesCommandUsernameDescription"/>
       <SingleValueOption Name="password" Shortcut="p" Help="SourcesCommandPasswordDescription"/>
       <SwitchOption Name="store-Password-In-Clear-Text" Help="SourcesCommandStorePasswordInClearTextDescription"/>
       <SingleValueOption Name="valid-Authentication-Types" Help="SourcesCommandValidAuthenticationTypesDescription" />
       <SingleValueOption Name="configfile" Help="Option_ConfigFile" />
+      <Example Title="Add `nuget.org` as a source:" Command="dotnet nuget add source https://api.nuget.org/v3/index.json -n nuget.org" />
+      <Example Title="Add `c:\packages` as a local source:" Command="dotnet nuget add source c:\packages" />
+      <Example Title="Add a source that needs authentication:" Command="dotnet nuget add source https://someServer/myTeam -n myTeam -u myUsername -p myPassword --store-password-in-clear-text" />
+      <Example Title="Add a source that needs authentication (then go install credential provider):" Command="dotnet nuget add source https://azureartifacts.microsoft.com/myTeam -n myTeam" />
+      <SeeAlso Title="Package source sections in NuGet.config files" Url="/nuget/reference/nuget-config-file#package-source-sections" />
```

Comment:
By prefixing all these URLs with `/`, you're telling the docs validator to not validate the link.

https://github.com/NuGet/docs.microsoft.com-nuget#links

> Relative links to other pages in this docset should be in the form `../<folder>/<topic-file>.md` including the trailing .md.

Reply:
is dotnet and nuget the same docset?
the lint tool and the review by the sdk docs folks didn't raise an issue here.

Reply:
the linter isn't smart enough to warn that a link starting with `/` is from the current docset, so no, it's not going to warn you.

But, I forgot that  the dotnet cli docs live in the dotnet docs repo and not ours. In that case, yes, we need to keep that `/` prefix.

Reply:
@zivkan - cool. can you approve then please?